aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -160,6 +194,27 @@ define(["dojo/_base/declare",
       folderImage: "folder-64.png",
 
       /**
+       * Indicates whether or not the thumbnail image should be given a shadow effect.
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      hasShadow: false,
```

Comment:
Yes.... the good reason is that this is a declarative framework and we need to give developers options without resorting to writing custom CSS and/or custom widgets.